doCNA/Run.py

```diff
@@ -42,7 +42,7 @@ def analyze (self):
                 self.get_ai ()
                 self.get_coverage ()
                 self.solve_windows ()
-            except:
+            except (AssertionError):
```

Comment:
FYI (not something that needs changed) single exceptions don't need to be wrapped in a tuple.
```
except AssertionError:
...
```
is fine here.